backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -59,4 +59,16 @@ package object backend {
   }
 
   case class PreemptedException(msg: String) extends Exception(msg)
+
```

Comment:
What's going on here? Are these supposed to extend BID? Is there a reason why BID isn't sealed? Or does this just suspiciously look ADT-ish but ain't?